services/movie.py

```diff
@@ -0,0 +1,41 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
```

Comment:
it is Optional[list[int]] because accepts default as None
Fix in other places(with all types)